### ISSUE TITLE
NTR: Mitochondrial adenosyltransferase deficiency in fibroblasts (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8194]
New term request Mitochondrial adenosyltransferase deficiency in fibroblasts (segal_070724224533):

**Comment by pnrobinson** [comment]
@MickeySegal
I cannot find exactly this label, but think it is probably

[Reduced hepatic methionine adenosyltransferase activity HP:0034693](https://hpo.jax.org/browse/term/HP:0034693)

If this is not so, please provide context.

**Comment by MickeySegal** [comment]
It was for https://omim.org/entry/251110 using an old source, but not mentioned in the OMIM entry so not likely to be a current clinical test.  I'm deleting this finding.
